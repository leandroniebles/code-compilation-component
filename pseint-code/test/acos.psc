Proceso sin_titulo
	escribir acos(1);
	escribir acos(-1);
	escribir acos(cos(0));
	escribir acos(cos(pi));
	escribir acos(2);
FinProceso
