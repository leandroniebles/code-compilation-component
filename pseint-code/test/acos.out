0
3.1415926536
0
3.1415926536
=== Line 6: ExeError 312
